lib/jormungandr/test/integration/Cardano/LauncherSpec.hs

```diff
@@ -12,12 +12,15 @@ import Control.Concurrent.Async
     ( async, cancel, race, wait )
 import Control.Monad
     ( void )
+import System.Directory
+    ( removePathForcibly )
 import Test.Hspec
     ( Spec, describe, expectationFailure, it )
 
 spec :: Spec
 spec = describe "cardano-wallet-launcher" $ do
     it "Can start launcher against testnet" $ do
+        removePathForcibly "/tmp/cardano-wallet-jormungandr"
```

Comment:
Should we remove the storage from the Jörmungandr config, to use in-memory instead?

Reply:
Hmmmm. No. Ideally integration tests reproduce the production setup as close as possible. This means, among othert things, using file based storage.